### DISCUSSION TITLE
How to build 4.4 on ubuntu 20.04?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [audacious-media-player/audacious#1457] [Q&A]
I installed qt6 to a folder using aqtinstall. How can I get ./configure to recognise it? I suppose I might need to do something to get the compiled binary to recognise it too.

**Comment by radioactiveman** [comment]
Audacious 4.4 still supports Qt 5, see `./configure --help`. It should be easier to build Audacious for that Qt version on such old distributions.

For Qt 6 you need a recent Qt version which ships pkgconfig (.pc) files. And depending on the location where they are installed, you may then need to adjust the environment variable `PKG_CONFIG_PATH`.

**Reply by mariomadproductions** [reply]
Just to double check - is Qt5 and Qt6 support the same? No features missing/incomplete functionality with one or the other?

**Reply by radioactiveman** [reply]
Yes, there is no difference in terms of features/functionality from Audacious' point of view.
